src/test/java/duke/StorageTest.java

```diff
@@ -0,0 +1,24 @@
+package duke;
```

Comment:
Neat code throughout the project in general. Code style only differs very slightly from the required. Good job!

Reply:
Thanks a lot for all your suggestions and catches!